### DISCUSSION TITLE
Cant locate object detection API for fast RCNN

### DISCUSSION THREAD

**Discussion by SudhaBhyri** [community/community#63699] [Repositories]
### Select Topic Area

General

### Body

Hi,

I am unable to locate an API for implementing fast RCNN anywhere! Its Faster RCNN everywhere. Could you please help me point to the right directory which has the pre-trained models for fast RCNN. I want to use is it for my project related to object detection for Stanford Cars Dataset. I have a submission deadline of 18th August. So its an urgent request. This is the first time I am posting anything here so I am not sure what I should expect. 

Really looking forward for a response.

Regards,
Sudha

**Comment by nirav-gajera** [comment]
- The original Fast RCNN paper: https://arxiv.org/abs/1504.08083
- The Fast RCNN GitHub repository: https://github.com/rbgirshick/fast-rcnn
- The PyTorch Object Detection API: https://github.com/facebookresearch/Detectron2 (includes a Fast RCNN implementation)
- The TensorFlow Object Detection API: https://github.com/tensorflow/models/tree/master/research/object_detection (includes a Fast RCNN implementation)
